leshan-client-core/src/main/java/org/eclipse/leshan/client/resource/ObjectEnabler.java

```diff
@@ -35,23 +39,28 @@
 import org.eclipse.leshan.core.request.BootstrapWriteRequest;
 import org.eclipse.leshan.core.request.CreateRequest;
 import org.eclipse.leshan.core.request.DeleteRequest;
+import org.eclipse.leshan.core.request.DiscoverRequest;
 import org.eclipse.leshan.core.request.ExecuteRequest;
 import org.eclipse.leshan.core.request.ObserveRequest;
 import org.eclipse.leshan.core.request.ReadRequest;
+import org.eclipse.leshan.core.request.WriteAttributesRequest;
 import org.eclipse.leshan.core.request.WriteRequest;
 import org.eclipse.leshan.core.request.WriteRequest.Mode;
 import org.eclipse.leshan.core.response.BootstrapWriteResponse;
 import org.eclipse.leshan.core.response.CreateResponse;
 import org.eclipse.leshan.core.response.DeleteResponse;
+import org.eclipse.leshan.core.response.DiscoverResponse;
 import org.eclipse.leshan.core.response.ExecuteResponse;
 import org.eclipse.leshan.core.response.ObserveResponse;
 import org.eclipse.leshan.core.response.ReadResponse;
+import org.eclipse.leshan.core.response.WriteAttributesResponse;
 import org.eclipse.leshan.core.response.WriteResponse;
 
 public class ObjectEnabler extends BaseObjectEnabler {
 
     private Map<Integer, LwM2mInstanceEnabler> instances;
     private LwM2mInstanceEnablerFactory instanceFactory;
+    private AttributeSet objectAttributes;
```

Comment:
I'm having trouble seeing why there is both an ObjectEnabler and a BaseObjectEnabler, maybe they could be merged in a later PR? I put the attributes in the ObjectEnabler since it seemed that the most logic already was placed there, and some protected methods could be of use.